src/pudl/metadata/resources/eia860.py

```diff
@@ -476,6 +476,36 @@
         "sources": ["eia860", "eia923"],
         "etl_group": "eia860",
     },
+    #     "emissions_control_equipment_eia860": {
+    #         "description": (
+    #             """
+    # The cost, type, operating status, retirement date, and install year of emissions control
+    # equipment reported to EIA. Includes control ids for sulfur dioxide (SO2), particulate
+    # matter, mercury, nitrogen oxide (NOX), and acid (HCl) gas monitoring.
+    # """
+    #         ),
+    #         "schema": {
+    #             "fields": [
+    #                 "report_year",
+    #                 "plant_id_eia",
+    #                 "emission_control_id_pudl",
+    #                 "data_maturity",
+    #                 "emission_control_equipment_type",
+    #                 "operational_status_code",
+    #                 "mercury_control_id_eia",
+    #                 "nox_control_id_eia",
+    #                 "particulate_control_id_eia",
+    #                 "so2_control_id_eia",
+    #                 "acid_gas_control",
+    #                 "emission_control_equipment_cost",
```

Comment:
I excluded fields I know are foreign but not part of the primary key including: `plant_name_eia`, `utility_id_eia`, `utility_name_eia`, and `steam_plant_type_code`. I'm assuming that not including them here means they are not included in the final version of the table. This assumption means that I don't alter or drop these columns in the transform module. Does that make sense?

Reply:
that does make sense. if you don't want these columns in the published table, the enforce_schema step - which is always applied - will drop them. all of transformed assets made in transform.eia860 are fed into the harvesting process automatically so the values from this table are being slurped up then. which is all good.